docs/gethue/content/en/posts/2021-06-14-release-hue-4-10.md

```diff
@@ -0,0 +1,132 @@
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.10
+  - Release
+
+---
+Hi Data Explorers,
+
+The Hue Team is glad to thanks all the contributors and release Hue 4.10!
+
+<a href="https://cdn.gethue.com/uploads/2021/02/hue-4.9.png">
+  <img src="https://cdn.gethue.com/uploads/2021/02/hue-4.9.png" />
+</a>
+
+&nbsp;
+
+Here is a summary of the [main improvements](/categories/version-4.10/) of 4.10 on top of the previous [4.9](/blog/hue-4-9-sql-dialects-phoenix-dasksql-flink-components/) release:
+
+
+## SQL Editor Component & API
+Now build your own SQL Editor in two HTML line by leveragin the [SQL Scratchpad](https://docs.gethue.com/developer/components/scratchpad/) component. The first version of the public [REST API](https://docs.gethue.com/developer/api/rest/) for executing queries was published to.
+
+Read more about the [Sql Scratchpad and REST API](https://gethue.com/blog/2021-05-29-create-own-sql-editor-via-webcomponent-and-public-api/).
+
+Here is a live demo of how easy it is to add the component:
+
+{{< rawhtml >}}
+<p>
+  <div style="position: absolute; height: 10%; width: 100%">
+    <sql-scratchpad api-url="https://demo.gethue.com" username="demo" password="demo" dialect="mysql" />
+  </div>
+
+  <script type="text/javascript" src="https://unpkg.com/gethue/lib/components/SqlScratchpadWebComponent.js"></script>
+
+  <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
+</p>
+{{< /rawhtml >}}
+
+![Adding the component in 3 lines and watching the interaction with the public API of demo.gethue.com](https://cdn-images-1.medium.com/max/2356/1*yXRjYQN_eRUimzlXPl5SwQ.gif)*Adding the component in 3 lines and watching the interaction with the public API of demo.gethue.com*
+
+## Create Table Wizard
+
+Create a Hive, Impala, MySql, Phoenix SQL table via a small file in three clicks. Just upload it via your browser, without the need to have any Storage filesystem like HDFS or S3 configured and follow the wizard.
+
+Read more about the [create table from a small file](https://gethue.com/blog/2021-05-26-improved-hue-importer-select-a-file-choose-a-dialect-create-a-table/).
+
+![Importer direct upload steps](https://cdn.gethue.com/uploads/2021/05/Importer_direct_upload_steps.gif)
+
+## Slack App
+Collaborate more via Slack thanks to the rich preview of SQL queries, auto links, query bank and export result to Slack.
+
+[Install the app](/blog/2021-05-18-installing-hue-slack-app-in-three-simple-steps/) in a few clicks and learn more about the [functionalities of assistance](https://docs.gethue.com/user/concept/#slack).
+
+![Slack Installation Flow](https://cdn.gethue.com/uploads/2021/05/slack-install.gif)
```

Comment:
This gif is also present in the `Install the app` blog. Can we show the easier installation gif? Something like

`Can also be installed within seconds by just giving your Hue hostname and  following the [steps] (https://docs.gethue.com/administrator/configuration/server/#improved-slack-app-installation)`

```![Slack Installation Flow](https://cdn.gethue.com/uploads/2021/06/easier_installation_slack.gif)```